source/features/notifications-navigation.tsx

```diff
@@ -0,0 +1,25 @@
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import features from '../feature-manager';
+import observe from '../helpers/selector-observer';
+
+function addNextPreviousBtn(navigationButtons: Element): void {
+	select('notification-filter')!.closest('.d-flex')!.classList.add('flex-justify-between');
+	const nextPreviousBtns = navigationButtons.cloneNode(true);
+	nextPreviousBtns?.classList.add('d-none', 'ml-3', 'd-lg-flex');
+	if (nextPreviousBtns) {
+		select('notification-filter')!.closest('.d-flex')!.append(nextPreviousBtns);
+	}
+}
+
+async function init(signal: AbortSignal): Promise<void> {
+	observe('.js-notifications-list-paginator-buttons', addNextPreviousBtn, {signal});
```

Comment:
Can you verify that the feature works correctly when navigating back and forth through the pages?

- click next multiple times, click prev to go back
- then go "back" with the browser "back" button, then forward again 
- then try marking 20+ the notifications as done so that the list refreshes


 The buttons should be updated correctly in every case.